sisl/geom/bilayer.py

```diff
@@ -0,0 +1,95 @@
+import numpy as np
+
+from sisl import Atom, geom, Cuboid
+
+__all__ = ['bilayer']
+
+
+def bilayer(bond, atom=None, twist=(5, 6), separation=3.35, return_angle=False, layer='both'):
+    """ Commensurate unit cell of a twisted (hexagonal) bilayer structure.
+
+    This routine follows the prescription of twisted bilayer graphene found in
+    Laissardiere et al., Nano Lett. 10, 804-808 (2010).
+
+
+    Parameters
+    ----------
+    bond : float
+       length between atoms in nano-tube
+    atom : Atom(6)
+       bilayer atoms
+    twist : (int, int)
+       coordinates (m, n) defining the twist angle (theta)
+    separation : float
+       distance between the two layers
+    return_angle : bool
+       computed twist angle is returned in addition to the geometry instance
```

Comment:
In what unit is `angle`?

Reply:
Fixed in f5695c859bb205eae43694de62b3b4433b435b9e